watertap/unit_models/zero_order/tests/test_CANDOP_zo.py

```diff
@@ -176,12 +173,15 @@ def test_solution(self, model):
             value(model.fs.unit.properties_byproduct[0].conc_mass_comp["phosphates"])
             < 1e-6
         )
-        assert pytest.approx(500, rel=1e-5) == value(
-            model.fs.unit.properties_byproduct[0].conc_mass_comp[
-                "bioconcentrated_phosphorous"
-            ]
+        assert (
+            value(
+                model.fs.unit.properties_byproduct[0].conc_mass_comp[
+                    "bioconcentrated_phosphorous"
```

Comment:
Is there a reason why pytest.approx is not used in this check?

Reply:
I am just trying to make sure that the BCP in the byproduct stream is ~0. I used a < constraint just to make sure that the BCP in that stream is very small, and `pytest.approx` is only meant to be used for equality constraints..